src/account/Support.test.tsx

```diff
@@ -12,7 +12,7 @@ import { createMockStore, getMockStackScreenProps } from 'test/utils'
 const renderSupport = () =>
   render(
     <Provider store={createMockStore()}>
-      <Support {...getMockStackScreenProps(Screens.Support)} />
```

Comment:
Since we weren't passing `isTabNav: true`, this was actually using the SupportDrawer screen, so snapshot was looking to match that.